docs/web3.eth.rst

```diff
@@ -1017,9 +1021,10 @@ The following methods are available on the ``web3.eth`` namespace.
             "gas": "21000"
         }
 
-    The method ``eth_createAccessList`` returns list of addresses and storage keys used by the transaction, plus the gas consumed when the access list is added.
-
-    That is, it gives you the list of addresses and storage keys that will be used by that transaction, plus the gas consumed if the access list is included. Like ``eth_estimateGas``, this is an estimation; the list could change when the transaction is actually mined. Adding an ``accessList`` to your transaction does not necessary result in lower gas usage compared to a transaction without an access list.
+    The method ``eth_createAccessList`` returns a list of addresses and storage keys used by the transaction, plus the gas
+    consumed when the ``accessList`` is included. Like ``eth_estimateGas``, this is an estimation; the list could change when
+    the transaction is actually mined. Adding an ``accessList`` to your transaction does not necessary result in lower
```

Comment:
1. Let's steer away from "mining". Blocks are considered "finalized" when they earn the highest degree of confidence in consensus. Perhaps we adopt that word here? Or "... included in a block."?

2. necessary -> necessarily

```suggestion
    the transaction is finalized. Adding an ``accessList`` to your transaction does not necessarily result in lower
```